Remove references to manual pages not delivered by Solaris and update the
resolver(5) reference to resolv.conf(5).

This patch was developed in-house. Since it is Solaris-specific it is not
suitable for upstream.

--- dnsmasq-2.93/man/dnsmasq.8
+++ dnsmasq-2.93/man/dnsmasq.8
@@ -617,3 +617 @@
-These IP sets must already exist. See
-.BR ipset (8)
-for more details.
+These IP sets must already exist.
@@ -624,3 +622 @@
-These sets must already exist. See
-.BR nft (8)
-for more details. The family, table and set are passed directly to the nft. If the spec starts with 4# or 6# then
+These sets must already exist. The family, table and set are passed directly to the nft. If the spec starts with 4# or 6# then
@@ -2836 +2832 @@
-.BR resolver (5)
+.BR resolv.conf (5)
--- dnsmasq-2.93/man/es/dnsmasq.8
+++ dnsmasq-2.93/man/es/dnsmasq.8
@@ -1518 +1518 @@
-.BR resolver (5)
+.BR resolv.conf (5)
--- dnsmasq-2.93/man/fr/dnsmasq.8
+++ dnsmasq-2.93/man/fr/dnsmasq.8
@@ -517,2 +517 @@
-même façon que pour --address. Ces groupes d'IP doivent déjà exister. Voir
-ipset(8) pour plus de détails.
+même façon que pour --address. Ces groupes d'IP doivent déjà exister.
@@ -2368 +2367 @@
-.BR resolver (5)
+.BR resolv.conf (5)
--- dnsmasq-2.93/man/sv/dnsmasq.8
+++ dnsmasq-2.93/man/sv/dnsmasq.8
@@ -616,3 +616 @@
-Dessa IP-uppsättningar måste redan existera. Se
-.BR ipset (8)
-för mer information.
+Dessa IP-uppsättningar måste redan existera.
@@ -623,3 +621 @@
-Dessa uppsättningar måste redan existera. Se
-.BR nft (8)
-för mer information. Familjen, tabellen och uppsättningen skickas direkt till nft. Om specifikationen börjar med 4# eller 6# läggs
+Dessa uppsättningar måste redan existera. Familjen, tabellen och uppsättningen skickas direkt till nft. Om specifikationen börjar med 4# eller 6# läggs
@@ -2867 +2863 @@
-.BR resolver (5)
+.BR resolv.conf (5)
